documentation/howto/maven_basics.adoc

```diff
@@ -0,0 +1,147 @@
++
+.Select a Module Archetype
+image::mvnbasic_select_module_archetype.png[Select a Module Archetype]
++
+.. Click `Next` to continue.
+. In the third step of the `New Maven Module` wizard, enter the archetype details:
+.. Add a unique group ID to the `Group Id` field. Note that this ID must be different from the group ID of the included Maven project.
+.. Add the desired version number in the `Version` field. As an example, the version `0.0.1-SNAPSHOT` was used as the version number for the previously created Maven project, therefore here we can use a variation such as `0.1.1-SNAPSHOT`.
++
+.Set the Module Archetype Parameters
+image::mvnbasic_archetype_parameters.png[Set the Module Archetype Parameters]
++
+.. Click `Finish` to conclude setting up the Maven module.
+. (Optional) To change the settings for the created Maven module, expand the module name in the `Project Explorer` view and double click `pom.xml` from the expanded list. An `Overview` tab appears and all settings can be changed from this tab.
++
+.Change the Module Settings from the Overview View
+image::mvnbasic_module_pom_overview.png[Change the Module Settings from the Overview View]
++
+
+**Result:** Your new Maven module is created and appears in the `Project Explorer` view.
+
+
+[[Maven_Support]]
+== 3. Add Maven Support to an Existing Non-Maven Project
+
+The previous tasks contain instructions to create a new Maven project and Maven module. However, for an existing application that was not created with Maven support, use the following instructions to add Maven support to the non-Maven project:
+
+==== Adding Maven Support to an Existing Non-Maven Project ====
+. (Optional) To add dependencies to the project, use the following instructions:
+.. Right-click the project name in the `Project Explorer` view and click `Properties`.
+.. In the `Java Build Path` window, click `Libraries`.
++
+.Set the Java Build Path
+image::mvnbasic_java_build_path.png[Set the Java Build Path]
++
+.. Click `Add External JARs`, navigate to the required JAR file and select it. The new JAR will appear in the list in the `Libraries` tab.
+. Right-click the project name in the `Project Explorer` view.
+. From the displayed options, click `Configure`.
+. From the displayed sub-menu, click `Convert to Maven Project`.
+. Configure details for the new pom file:
+.. The basic fields for the new pom file are prepopulated based on the project details. If required, edit the existing values.
+.. (Optional) Add a name for the new project in the `Name` field.
+.. (Optional) Add a brief description for the project in the `Description` field.
++
+.Create a New Pom Descriptor
+image::mvnbasic_create_new_pom.png[Create a New Pom Descriptor]
++
+.. Click `Finish` to finalize the pom information.
+. If dependencies were added (using the instructions in step 1), a wizard appears displaying all added dependencies and a green check mark when each dependency is identified.
```

Comment:
should it not explain how this identification is made and what to do in the case dependencies are not matched?